eng/common/scripts/Helpers/Package-Helpers.ps1

```diff
@@ -170,10 +170,86 @@ function GetValueSafelyFrom-Yaml {
         $current = $current[$key]
       }
       else {
-        Write-Host "The '$key' part of the path $($Keys -join "/") doesn't exist or is null."
```

Comment:
@JimSuplizio FYI

This is very spammy when invoking for `MatrixConfigs`, as it is mostly null. We're returning `$null`, so the caller can output if they want to.